thunderbird/MozillaThunderbird.spec

```diff
@@ -330,6 +328,9 @@ export CFLAGS="$CFLAGS -mminimal-toc"
 %endif
 %endif
 export CXXFLAGS="$CFLAGS"
+%ifarch %{arm} aarch64
+export RUSTFLAGS="-Cdebuginfo=0"
+%endif
```

Comment:
Here I would like to have a comment, why this is needed, and why only for aarch64. And under what circumstances it could be removed again.

Reply:
good question. The section was already in Thunderbird 60 and was inherited from Firefox 60. It was not in the SLE version of FF60? I'm happy to remove it. I'm sure it's didn't come from my side anyway looking at the architectures if you agree.

Reply:
aarch64 builds fine without this on SLE-15 and SLE-12 (I'm assuming this was to fix a build)

Reply:
so let's remove it and remember if it fails somewhere again